tests/component/test_timing_properties.py

```diff
@@ -0,0 +1,221 @@
+"Contains a collection of pytest tests that validates the timing properties."
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx import DaqError
+from nidaqmx.constants import AcquisitionType, SampleTimingType
+from nidaqmx.error_codes import DAQmxErrors
+
+
+def test__timing__get_boolean_property__returns_value(any_x_series_device):
+    """Test for validating getter for boolean property."""
+    with nidaqmx.Task() as task:
+        task.ao_channels.add_ao_voltage_chan(any_x_series_device.ao_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.ON_DEMAND
+
+        assert not task.timing.simultaneous_ao_enable
+
+
+def test__timing__set_boolean_property__returns_assigned_value(any_x_series_device):
+    """Test for validating setter for boolean property."""
+    with nidaqmx.Task() as task:
+        task.ao_channels.add_ao_voltage_chan(any_x_series_device.ao_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.ON_DEMAND
+
+        task.timing.simultaneous_ao_enable = True
+
+        assert task.timing.simultaneous_ao_enable
+
+
+def test__timing__reset_boolean_property__returns_default_value(any_x_series_device):
+    """Test for validating reset for boolean property."""
+    with nidaqmx.Task() as task:
+        task.ao_channels.add_ao_voltage_chan(any_x_series_device.ao_physical_chans[0].name)
+        task.timing.samp_timing_type = SampleTimingType.ON_DEMAND
+
+        del task.timing.simultaneous_ao_enable
```

Comment:
In the arrange block, set `simultaneous_ao_enable = True`. If you don't set the property, reset has no effect.